src/state/shellscripts/sandbox_exec.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -ue
+#!/bin/bash -ue
```

Comment:
FWIW, I think the posix[tm] way to do this is `#!/usr/bin/env bash` -- this may even lead to success on non-GNU/Linux systems that have bash installed optionally.

Reply:
Thanks, I proposed this change in #3405.

Reply:
Discussing this in the other PRs; in this case it's OSX, though, which should always have `bash` in `/bin` ?